tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -31,15 +32,21 @@
 
 using namespace ABI::Windows::ApplicationModel::Activation;
 using namespace ABI::Windows::Media::SpeechRecognition;
+using namespace ABI::Windows::Foundation;
 using namespace Microsoft::WRL;
 
+// Method to call in tests to activate app
+extern "C" void UIApplicationActivationTest(IInspectable* args);
+
+// These are for testing that delegate methods are actually called
+static NSMutableDictionary* cortanaVoiceCommandForegroundDelegateMethodsCalled = [NSMutableDictionary new];
```

Comment:
nit: Need not be static, can be part of CortanaXXXForegroundTestDelegate test interface